etcdctl/ctlv3/command/move_leader_command.go

```diff
@@ -19,6 +19,7 @@ import (
 	"strconv"
 
 	"github.com/spf13/cobra"
+
```

Comment:
minor comment: Did you intentionally add this blank line or it's added by IDE automatically? I would suggest to keep it as it's. Of course, it isn't a big  deal.

Reply:
+1

Reply:
yep IDE, let me remove it again